src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -264,7 +265,9 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
       <TopBackground style={{ height: "100%", maxHeight: "auto" }} />
       <Card>
         <ActivityIndicator size="large" color={color("grey", 40)} />
-        <AppText style={{ marginTop: size(1) }}>Checking...</AppText>
+        <AppText style={{ marginTop: size(1) }}>
+          {i18n.t("customerQuotaScreen.quotaCheck")}...
+        </AppText>
       </Card>
     </View>
   ) : (
```

Comment:
You missed out a few copies on `ItemsSelectionCard`, e.g. Checkout, Cancel, Back (on scan screen)